src/golang/lib/models/shared/execution_status.go

```diff
@@ -23,7 +23,11 @@ const (
 	CanceledExecutionStatus   ExecutionStatus = "canceled"
 	FailedExecutionStatus     ExecutionStatus = "failed"
 	SucceededExecutionStatus  ExecutionStatus = "succeeded"
-	UnknownExecutionStatus    ExecutionStatus = "unknown"
+	// 'erased' refers to 'erased after success'.
```

Comment:
What is "erased" here mean?

Reply:
I think here is a good place to lay out what the lifecycle is, with an ascii diagram:
Registered -> Pending -> Running -> Failed
 .......................................................-> Cancelled
 .......................................................-> Succeeded -> Deleted [Optional]